plugins/janus_recordplay.c

```diff
@@ -562,16 +597,19 @@ static const char *janus_recordplay_parse_codec(const char *dir, const char *fil
 			offset += 2;
 			if(len > 0 && !parsed_header) {
 				/* This is the info header */
-				bytes = fread(prebuffer, sizeof(char), len, file);
+				char *json_header = malloc(sizeof(char)*(len+1));
```

Comment:
Please use `g_malloc`/`g_free`. That said, not sure why you're allocating? 1500 bytes should be more than enough even if extensions are in the json.

Reply:
In theory it's possible (even though it's highly unlikely) for JSON header to have more than 1500 chars if there are many extmaps with long URIs.